Modules/core/View/Coretiles/welcomeAction.php

```diff
@@ -209,7 +209,7 @@
                             "resource": elem.resource,
                             "id_resource": elem.resource_id,
                             "space": elem.space,
-                            "date": `${bdate.toLocaleDateString()} ${bdate.toLocaleTimeString()}`
+                            "date": `${bdate.toLocaleDateString()} ${bdate.getUTCHours()}:${bdate.getMinutes()}`
```

Comment:
Doesn't force 2 digits ("17:00" will be displayed as "17:0" for example).
Found that on github: `(date.getMinutes()<10?'0':'') + date.getMinutes()`. Should solve the problem